includes/sanitizers/class-amp-auto-lightbox-disable-sanitizer.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Class AMP_Auto_Lightbox_Disable_Sanitizer
+ *
+ * @package AmpProject\AmpWP
+ */
+
+/**
+ * Disable auto lightbox for images.
+ *
+ * @since 2.2.2
+ * @internal
+ */
+class AMP_Auto_Lightbox_Disable_Sanitizer extends AMP_Base_Sanitizer {
+
+	/**
+	 * Add "data-amp-auto-lightbox-disable" attribute to body tag.
+	 *
+	 * @return void
+	 */
+	public function sanitize() {
+		$this->dom->body->setAttributeNode( $this->dom->createAttribute( 'data-amp-auto-lightbox-disable' ) );
```

Comment:
It's strange. The `sanitize` function is not executed on my local environment and thus the `body` tag is not assigned the `data-amp-auto-lightbox-disable` attribute.

On the other hand, the sanitizer is correctly added to the list in [`amp_get_content_sanitizers`](https://github.com/ampproject/amp-wp/pull/6936/files#diff-4d201f41a17fa3b698d9548b8d417866cb3f03780acd8692b7accac477efeb65R1614).

Reply:
> It's strange. The sanitize function is not executed on my local environment and thus the body tag is not assigned the data-amp-auto-lightbox-disable attribute.

I think it could be because of auto-loader file is not updated. Can be checked if the error log has an error similar to below.

```
PHP Notice:  AMP_Content_Sanitizer::sanitize_document was called <strong>incorrectly</strong>. Sanitizer (AMP_Auto_Lightbox_Disable_Sanitizer) class does not exist Please see <a href="https://wordpress.org/support/article/debugging-in-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 0.4.1.) in /Users/dhavalparekh/Sites/amp-dev/wp-includes/functions.php on line 5768
```

Try after removing and re-installing `vendors` directory.

Reply:
Thanks, it worked.

Reply:
Thanks, it worked.